uber/utils.py

```diff
@@ -477,6 +477,31 @@ def get_real_badge_type(badge_type):
     return c.ATTENDEE_BADGE if badge_type in [c.PSEUDO_DEALER_BADGE, c.PSEUDO_GROUP_BADGE] else badge_type
 
 
+def add_opt(opts, other):
+    """
+    Add an option to a comma-separated string.
+    """
+    other = [other] if isinstance(other, str) else other
+    list = opts.split(',') if opts else []
+    list.extend(other)
+    return ','.join(list)
+
+
+def remove_opt(opts, other):
+    """
+    Remove an option from a comma-separated string.
+    """
+    other = [other] if isinstance(other, str) else other
+    list = opts.split(',')
+    if other == list:
```

Comment:
Did you know the `if x in y` will return false for lists if `x == y`? I sure didn't!

Unit tests caught this problem. Hooray for unit tests!